src/System.Private.Xml/tests/XmlSerializer/ReflectionOnly/System.Xml.XmlSerializer.ReflectionOnly.Tests.csproj

```diff
@@ -7,6 +7,9 @@
   </PropertyGroup>
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'netstandard-Debug|AnyCPU'" />
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'netstandard-Release|AnyCPU'" />
+  <ItemGroup Condition="'$(TargetGroup)' == 'netcoreapp'">
+    <Compile Include="$(TestSourceFolder)..\XmlSerializerTests.netcoreapp.cs" />
```

Comment:
No, I just verified and it is not being included even when `"'$(TargetGroup)' == 'netcoreapp'"` because there is only a netstandard build configuration so TargetGroup will always be netstandard. I'm trying to figure out a workaround for this to leave the build configurations as they are.

I guess we want to have this tests included in uap right?